files/jira.rb

```diff
@@ -5,10 +5,15 @@
 class Jira < BaseHandler
 
   def build_labels
-    [ "SENSU_#{@event['client']['name']}",
+    [
+      "SENSU_#{@event['client']['name']}",
       "SENSU_#{@event['check']['name']}",
-      "SENSU", *@event['check']['tags'] ].uniq.reject { |x| x.nil? }.map { |x|
-        x.strip.gsub(/\s+/, '_') }
+      "SENSU",
+      *@event['check']['tags'],
+      *team_data('tags'),
```

Comment:
For contextual reasons, this is a goal of this change. The pattern that we see a lot is that folks have separate projects for components that typically have the same pagerduty api key, but separate JIRA projects and irc channels. We want to support having these kind of separate reporting styles but on a single JIRA project, so we need the ability to add arbitrary labels (tags) for reporting purposes.

For example, previously we cut tickets for Cassandra hosts on the CASSANDRA project and tickets for Elasticsearch hosts on the ES project, but now we'd like to cut tickets for both on the "Data Oncall" project and differentiate with labels.